Closes #261: Improve lando instructions in migration section of README (#262)
az-digital/az-quickstart-pantheon: README.md

Reviewer: ```suggestion
lando pull -d live -f live -c none
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
lando migrate-setup-from-pantheon -s <source-site-name>.<environment>
```

## Running migrations from Drupal 7 or UA Quickstart site downloaded from Pantheon, to Drupal 9 Arizona Quickstart site downloaded from Pantheon

> NOTE: The `lando migrate-setup-from-pantheon` command requires that a site exists on Pantheon, since it uses terminus to find site variables on the source site. Getting started with local Lando developement can be found [here.](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#local-development)

Reviewer 2: ```suggestion
NOTE: The `lando migrate-setup-from-pantheon` command requires that a site exists on Pantheon, since it uses terminus to find site variables on the source site.
NOTE: Instructions for how to use Lando for local development when contributing general fixes and enhancements forQuickstart (not Pantheon-specific) can be found [in the main AZ Quickstart repository](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#local-development).
```

> Create your local copy of the Pantheon **destination** site by initializing Lando inside and *empty* folder.

Reviewer 2: ```suggestion
Create your local copy of the Pantheon **destination** site by initializing Lando inside an *empty* folder.
```

Reviewer 3: Maybe it would be good to end this line with a colon, and do the same with "Or execute the bash commands" below?

```

> mkdir <destinationsitename>

Reviewer 3: For easier reading, maybe change these to have underscores or dashes between the words?

```suggestion
mkdir <destination_site_name>
```

cd <destinationsitename>
lando init --source pantheon
```

> Or execute the bash commands

Reviewer 2: ```suggestion
Or execute the bash commands:
```

```
mkdir <destinationsitename> && cd "$_"
lando init --source pantheon
```

This starts the interactive site creation tool that allows you to choose

> your destination site from a list of sites created on Pantheon. Select the branch you are working on from the dropdown. 

Reviewer 2: When I tried this it prompted me for a Pantheon account to login with and then presented me with a list of sites.  I never saw a branch option.

```suggestion
your destination site from a list of sites created on Pantheon. Select the site you are working on from the dropdown. 
```

> ## Running migrations from Drupal 7 or UA Quickstart site downloaded from Pantheon, to Drupal 9 Arizona Quickstart site downloaded from Pantheon

Reviewer: ```suggestion
## Migrating a Pantheon Drupal 7 or UA Quickstart site to Modern Arizona Quickstart
```

NOTE: The `lando migrate-setup-from-pantheon` command requires that a site exists on Pantheon, since it uses terminus to find site variables on the source site.
NOTE: Instructions for how to use Lando for local development when contributing general fixes and enhancements to Quickstart (not Pantheon site specific) can be found in the [contributing guide in the main AZ Quickstart repository](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#local-development).

Create your local copy of the Pantheon **destination** site by initializing Lando inside an *empty* folder:
```

> mkdir <destination_site_name>

Reviewer: ```suggestion
mkdir <destination-site-name>
```

> cd <destination_site_name>

Reviewer: Pantheon doesn't allow underscores in the site name.

> cd <destination_site_name>

Reviewer: ```suggestion
cd <destination-site-name>
```

> lando init --source pantheon

Reviewer: > Does lando init even work? In my experience I get errors when I do it this way.

>In my opinion a better way to tell people to get a copy of the repository is to clone using git.

It works if you add the name parameter.

```suggestion
lando init --source pantheon --name <destination-site-name>
```

```
Or execute the bash commands:

```

> mkdir <destination_site_name> && cd "$_"

Reviewer: ```suggestion
mkdir <destination-site-name> && cd "$_"
```

Open the dev site when the push is complete, and do a spot check

```

> terminus env:view <destination_site_name.env>

Reviewer: ```suggestion
terminus env:view <destination-site-name>.<environment>
```

```

**Always make a backup if overwriting live**

```

> terminus backup:create <destination_site_name.live>

Reviewer: ```suggestion
terminus backup:create <destination-site-name>.live
```

```

If all looks good, go ahead and deploy your migration to another environment,
overwriting that environment’s database and files.

```

> terminus env:clone-content <destination_site_name.dev> <target_env> --cc -y

Reviewer: ```suggestion
terminus env:clone-content <destination-site-name>.dev <target_env> --cc -y
```

```

Check the destination site.

```

> terminus env:view <destination_site_name.live>

Reviewer: ```suggestion
terminus env:view <destination-site-name>.live
```
